x/treasury/keeper/migrations.go

```diff
@@ -18,5 +64,8 @@ func NewMigrator(keeper Keeper) Migrator {
 // Migrate1to2 migrates from version 1 to 2.
 func (m Migrator) Migrate1to2(ctx sdk.Context) error {
```

Comment:
I know it applies when upgrading from v1.0.5 to v1.0.6
Is it possible to use the same version for migration?

Reply:
I think this will be upgraded in v2 of the chain and not v1.6.0

Reply:
We cannot use the same treasury version 1 due to new store is added, so a migration to treasury version 2 is still required

Reply:
I thought version 2 would already be when upgrading to v1.0.5. I'm curious about this part.
That is, I thought that `Migrate1to2` function would not be called in v1.0.6.

Reply:
oh, in v1.0.5, treasury is still version 1. Migrate1to2() only exists in main branch.